backend/src/main/kotlin/app/ehrenamtskarte/backend/freinet/util/FreinetApi.kt

```diff
@@ -0,0 +1,149 @@
+                logger.error("Freinet search person API error: $e")
+                throw FreinetApiNotReachableException()
+            }
+        }
+
+    fun createPerson(
+        firstName: String,
+        lastName: String,
+        dateOfBirth: String,
+        personalDataNode: JsonNode?,
+        userEmail: String,
+        project: String,
+    ): Boolean {
+        val addressArrayNode = personalDataNode?.get("value")?.findValueByNameNode("address")
+        val street = addressArrayNode?.findValueByName("street")
+        val postalCode = addressArrayNode?.findValueByName("postalCode")
+        val regionId =
+            addressArrayNode?.findValueByNameNode(
+                "location",
+            )?.asText()?.toIntOrNull() ?: throw RegionNotFoundException()
+        val email = personalDataNode.get("value")?.findValueByName("emailAddress")
+        val phone = personalDataNode.get("value")?.findValueByName("telephone")
+        val region = RegionsRepository.findByIdInProject(project, regionId)
+            ?: throw RegionNotFoundException()
+
+        val currentDateTime = LocalDateTime
+            .now()
+            .format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
+
+        val requestBody = """
+      {
+        "init": {
+          "apiVersion": "3.0",
+          "agencyID": "$agencyId",
+          "accessKey": "$accessKey",
+          "modul": "personen", 
+          "author": "TuerAnTuer",
+          "author_mail": "berechtigungskarte@tuerantuer.org"
+        },
+        "person": {
+          "user_status": "1",
+          "vorname": "$firstName",
+          "nachname": "$lastName",
+          "geburtstag": "$dateOfBirth",
+          "address": {
+            "1": {
+              "adress_strasse": "$street",
+              "adress_plz": "$postalCode",
+              "adress_ort": "${region.name}",
+              "adress_mail1": "$email",
+              "adress_tel_p": "$phone"
+            }
+          }
+        },
+        "protokoll": {
+          "1": {
+            "title": "Erstellung des Nutzers durch Ehrenamtskarte Bayern von $userEmail",
+            "date": "$currentDateTime"
+          }
+        }
+      }
+        """.trimIndent()
+
+        return runBlocking {
+            try {
+                val response = httpClient.request {
+                    url {
+                        protocol = URLProtocol.HTTPS
+                        this.host = this@FreinetApi.host
+                        path("/api/input/v3/personen")
+                    }
+                    method = HttpMethod.Post
+                    contentType(ContentType.Application.Json)
+                    setBody(requestBody)
+                }
+                val body = response.bodyAsText()
+                logger.info("Successfully created person in freinet: $body")
+                true
+            } catch (e: Exception) {
+                logger.warn("Error creating person in freinet", e)
+                false
```

Comment:
❓ is there any reason why not throwing here directly the exception?

Reply:
I don't remember what the problem was but now I changed it to:
`logger.devWarn("Error creating person in freinet $e")`

Reply:
🔧 this is not resolved @bahaaTuffaha 
i think you should throw here an `FreinetApiNotReachable` as you did in the searchPerson query